node_modules/.yarn-integrity

```diff
@@ -1,10 +0,0 @@
-{
```

Comment:
node_modules should be in our .gitignore

Reply:
So it should be there, but just .gitignore'd? @shanejonas I thought you were recommending its removal.

Reply:
Nothing that you need to do here, just needs to be done as a seperate thing